.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+- "8"
+sudo: false
+cache:
+- node_modules
+script:
+- npm test
+env:
+  global:
+  - NODE_ENV=test
+  - GOOGLE_CLIENT_EMAIL=dGVzdAo=
```

Comment:
are these fake?

Reply:
Yes.  They are the string "test" base64 encoded.

Reply:
even if that's the case I think this is better off set as an ENV in travis, just so noone is tempted to replace these or ask questions in the future

Reply:
Done in 8515e8d.